process-engine/documentation
# Feature/prod build

## What did you change?

I added a script that updates the gh-pages branch using the current develop

## How can others test the changes?

you can run the script, and see that when it's done that:

- You are on the gh-pages-branch
- The Documentation in that branch is updated
- you can serve the `index.html` and see the current documentation

## PR-Checklist

Please check the boxes in this list after submitting your PR:
